src/site/content/en/blog/smoothness/index.md

```diff
@@ -0,0 +1,563 @@
+      </td>
+      <td>Compositor only; the main thread had a visual update that affects
+      smoothness, but a previously stale frame arrived and was used instead</td>
+    </tr>
+    <tr>
+      <td><strong style="background:#FDF2D0;padding:0.25em;">
+        Partially presented</strong>
+      </td>
+      <td>Compositor only; without the desired main update, and the
+      compositor update has an animation that affects smoothness</td>
+    </tr>
+    <tr>
+      <td><strong style="background:#EECDCD;padding:0.25em;">
+        Partially presented</strong>
+      </td>
+      <td>Compositor only but the compositor update does not have an
+      animation that affects smoothness</td>
+    </tr>
+    <tr>
+      <td><strong style="background:#EECDCD;padding:0.25em;">
+        Dropped frame</strong>
+      </td>
+      <td>No update. There was no compositor update desired, and main was
+      delayed</td>
+    </tr>
+    <tr>
+      <td><strong style="background:#DE9D9B;padding:0.25em;">
+        Dropped frame</strong>
+      </td>
+      <td>A compositor update was desired, but it was delayed</td>
+    </tr>
+    <tr>
+      <td><strong style="background:#DE9D9B;padding:0.25em;">
+        Stale frame</strong>
+      </td>
+      <td>An update was desired, it was produced by the renderer, but the
+      GPU still did not present it before the vsync deadline</td>
+    </tr>
+  </table>
+</div>
+
+It's possible to turn these states into somewhat of a score. And perhaps one way
+to interpret this score is to consider it a _probability_ of being observable by
+the user. A single dropped frame may not be very observable, but a sequence of
+many dropped frames affecting smoothness in a row sure is!
+
+## Putting it all together: a Percent Dropped Frames metric
+
+While it can sometimes be necessary to dive deep into the state of each
+animation frame, it's also useful to just assign a quick "at a glance" score for
+an experience.
+
+Because frames may be _partially presented,_ and because even fully skipped
+frame updates may not actually _affect smoothness_, we want to focus less on
+just counting frames, and more on the _extent_ to which the browser is unable to
+provide _visually complete_ updates _when it mattered_.
+
+The mental model should move from:
+
+1. *Frames Per Second*, to
+2. Detecting missing and important animation updates, to
+3. *Percentage Dropped* over a given time period.
+
+Put another way, what matters is: the proportion of time waiting for important
+updates. We think this matches the natural way users experience the smoothness
+of web content in practice. So far, we have been using the following as an
+initial set of metrics:
+
+- **Average Percent Dropped:** for all non-idle animation frames throughout the
+  whole timeline
+- **Worst Case of Percent Dropped Frames:** as measured over 1 second sliding
+  windows of time.
+- **95th percentile of Percent Dropped Frames:** as measured over 1 second
+  sliding windows of time.
```

Comment:
Not a necessary change, but I think this would stand out more as a table with Metric/Definition columns.